exonum-java-binding/core/pom.xml

```diff
@@ -262,6 +270,33 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-resources-plugin</artifactId>
+        <executions>
+          <!--We need to copy libjava_bindings dynamic library to the <ejb-app-location>/lib/native directory-->
```

Comment:
We do that so that the application will work from target, correct? Do we do the same with _Java_ dependencies in `packaging` module for the same reason? If so, shall we move the latter to this module (and possibly simplify the `packaging` module)? Do I understand correctly, that if that's done, I could get an application working from target by just building core and its dependencies:

```sh
mvn package -pl exonum-java-binding-core -am
```

instead of (?) currently *required* (because of Java-dep-copying step in `packaging`):

```sh
mvn package # Build all modules
```

Reply:
We also need to copy time-oracle classes, so it can't be done in Core module. That's why we actually created Packaging module

Reply:
I see, that makes sense.